tests/unit/oidc/test_views.py

```diff
@@ -392,18 +456,16 @@ def test_mint_token_from_oidc_no_pending_publisher_ok(
         id="fakeprojectid",
         record_event=pretend.call_recorder(lambda **kw: None),
     )
-    publisher = pretend.stub(
```

Comment:
This was changed to use the factory due to the type check [here](https://github.com/pypi/warehouse/pull/14063/files#diff-00352684c58d85c29397c3a8b7a868b5a7846e75a3dfb6f67af1ab141483ff0bR146) and [here](https://github.com/pypi/warehouse/pull/14063/files#diff-00352684c58d85c29397c3a8b7a868b5a7846e75a3dfb6f67af1ab141483ff0bR220) which was added because of mypy errors  in the code after that type check.